src/nn/Model.js

```diff
@@ -28,6 +28,7 @@ export default class Model {
     this._eager = options.eager || false;
     this._supportedOps = options.supportedOps || new Set();
     this._isQuantized = false;
+    this._isSRModel = false;
```

Comment:
`_isSRModel` is too specific. Please rename it to something like `_hasUnsupportedOp`